cluster-http/src/main/scala/akka/discovery/DnsSrvServiceDiscovery.scala

```diff
@@ -1,9 +1,9 @@
 /*
- * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
  */
 package akka.discovery
 
-import akka.actor.ActorSystem
+import akka.actor.{ ActorSystem, ExtendedActorSystem }
 import akka.event.Logging
 import akka.io.AsyncDnsResolver.SrvResolved
```

Comment:
This class is from ru akka-dns, so I'm still confused about this dependency.

Reply:
Right, please look at, my statement in the standup was incorrect -- i assumed I don't use it because I forgot it has packages in the "wrong package". and grepped the codebase so did not notice any `ru.` code.

Indeed this is the russian one.

Please look at the latest commit here https://github.com/akka/akka-management/pull/60/commits/5e3d773c92a14b39e85ce45437e2e4f0757ea007 which makes it really work without the dependency, but with limitations that we can't discover the management port from DNS records